docs/cudf/source/guide-to-udfs.ipynb

```diff
@@ -37,7 +37,7 @@
     "- Within each block, many \"threads\" operate independently and simultaneously access their block-specific shared memory with low latency\n",
     "\n",
     "\n",
-    "This guide covers APIs that automatically handle dividing columns into chunks and assigning them into different GPU blocks for parallel computation (see [apply_chunks](https://rapidsai.github.io/projects/cudf/en/0.10.0/api.html#cudf.dataframe.DataFrame.apply_chunks) or the [numba CUDA JIT API](https://numba.pydata.org/numba-doc/dev/cuda/index.html) if you need to control this yourself)."
+    "This guide covers APIs that automatically handle dividing columns into chunks and assigning them into different GPU blocks for parallel computation (see [apply_chunks](https://docs.rapids.ai/api/cudf/0.10/api.html#cudf.core.dataframe.DataFrame.apply_chunks) or the [numba CUDA JIT API](https://numba.pydata.org/numba-doc/dev/cuda/index.html) if you need to control this yourself)."
```

Comment:
```suggestion
    "This guide covers APIs that automatically handle dividing columns into chunks and assigning them into different GPU blocks for parallel computation (see [apply_chunks](https://docs.rapids.ai/api/cudf/0.14/api.html#cudf.core.dataframe.DataFrame.apply_chunks) or the [numba CUDA JIT API](https://numba.pydata.org/numba-doc/dev/cuda/index.html) if you need to control this yourself)."
```

Should we rather update to `0.14`/`0.15`?

Reply:
whatever you guys think is best. I just tried to achieve parity in this PR. but i can update it. we can use the `stable` link as well to make sure it's always up to date. i.e. https://docs.rapids.ai/api/cudf/stable/api.html#cudf.core.dataframe.DataFrame.apply_chunks

let me know!

Reply:
Lets go with the `stable` links then

Reply:
pushed!